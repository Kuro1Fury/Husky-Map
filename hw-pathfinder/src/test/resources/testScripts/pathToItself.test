# Test the path finding from a node to itself

CreateGraph A

AddNode A n1

FindPath A n1 n1