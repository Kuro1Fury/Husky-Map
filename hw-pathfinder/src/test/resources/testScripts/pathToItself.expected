# Test the path finding from a node to itself

created graph A

added node n1 to A

path from n1 to n1:
total cost: 0.000